.azure-pipelines/client.yml

```diff
@@ -124,9 +149,16 @@ jobs:
       summaryFileLocation: '$(Build.SourcesDirectory)/coverage.xml'
       reportDirectory: '$(Build.SourcesDirectory)/htmlcov'
 
+  - task: ms.vss-governance-buildtask.governance-build-task-component-detection.ComponentGovernanceComponentDetection@0
+    displayName: 'Component Detection'
+    condition: and(eq(variables['os.name'],'Windows'),succeeded())
+
 - job: Test_Alpha_Python
   timeoutInMinutes: 90
   condition: eq(variables['long_running_tests'], True)
+
+
+- job: Special_Python_Distro_Tests
```

Comment:
In the Test stage, you still have the component governance present with `only windows` condition. Please remove it as it's already part of the build in the `analyze` step.